woocommerce/woocommerce
# update references to woocommerce.com to now reference woo.com

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:




 
We moved `woocommerce.com` to `woo.com`. This PR replaces URLs all over the repo. I searched for `woocommerce.com` and then manually went through all the results to ensure a) I'm not changing cases an automatic search and replace didn't account for and b) I *am* changing things that an automatic search and replace might have overlooked. Both were the case so while it was quite some manual work it gave the results I was expecting. 

[I excluded URLs](https://github.com/woocommerce/woocommerce/pull/41241/commits/0e488ad1c2bca03fd0e66c5f796ddbb085a662cb) pointing to the internal `woocommerce.com/wp-json/` API. As it's not user- or even external developer-facing we're not switching that one. 



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. due to the nature of this change they're all over the codebase, so for the initial code review I'd recommend ensuring that all CI checks pass and taking samples to ensure replaced URLs work
2. in a second step, possibly after merge, it could make sense to have GS go through Woo.com Marketplace purchasing / subscription flows to ensure those were unaffected



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

update references to woocommerce.com to now reference woo.com

#### Comment 

</details>
